webnn-tflite-delegate/README.md

```diff
@@ -20,4 +20,7 @@ const model = await loadTFLiteModel('./mobilenetv2.tflite', {
 });
 
 ```
-Before running model with this package, set the environment variables for OpenVINO on [Linux](https://docs.openvino.ai/2021.4/openvino_docs_install_guides_installing_openvino_linux.html#set-the-environment-variables) or [Windows](https://docs.openvino.ai/2021.4/openvino_docs_install_guides_installing_openvino_windows.html#step-3-configure-the-environment).
+Before running model with this package, set following environment variables:
```

Comment:
Please add a direct link of OpenVINO installer, e.g.,

https://registrationcenter-download.intel.com/akdlm/irc_nas/18320/w_openvino_toolkit_p_2021.4.752.exe